Template/{PROJECT}.xcodeproj/project.pbxproj

```diff
@@ -330,7 +330,7 @@
 			attributes = {
 				LastSwiftUpdateCheck = 0720;
 				LastUpgradeCheck = 0810;
-				ORGANIZATIONNAME = "John Sundell";
+				ORGANIZATIONNAME = "{ORGANIZATIONNAME}";
```

Comment:
Haha, I totally missed this! It wasn't a plan to take over the universe, I promise! 😜
Can I make a small nitpick: can we name it `ORGANIZATION` instead, to follow the naming convention that the other variables follow?